pages/linux/tzselect.md

```diff
@@ -0,0 +1,12 @@
+# tzselect
+
+> Interactively select timezone.
+> More information: <https://manned.org/tzselect>.
+
+- Open the interactive menu for timezone selection:
+
+`tzselect`
```

Comment:
This program doesn't actually set the timezone, and the current description kind of implies that it does.

```suggestion
> Interactively select a timezone.
> Note: this program doesn't actually set the timezone.
> More information: <https://manned.org/tzselect>.

- Open the interactive menu for timezone selection and print the selected timezone to `stdout`:

`tzselect`
```